microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -287,6 +287,7 @@ Microsoft Edge supports a variety of first-party enterprise security and complia
 * [Windows Information Protection](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip)
 * [Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview)
 * [Microsoft Endpoint Data Loss Prevention (DLP)](/microsoft-365/compliance/endpoint-dlp-learn-about)
+* [Microsoft Defender for Cloud Applications](/defender-cloud-apps/what-is-defender-for-cloud-apps)
```

Comment:
```suggestion
* [Microsoft Defender for Cloud App](/defender-cloud-apps/what-is-defender-for-cloud-apps)
```
https://learn.microsoft.com/en-us/defender-cloud-apps/what-is-defender-for-cloud-apps
Actual destination page title: "Microsoft Defender for Cloud Apps overview"
# hits on "cloud applications" in that page: 0
# hits on "cloud apps" in that page: 34